### ISSUE TITLE
Only some Monday boards show in the form settings

### ISSUE THREAD

**Issue by LoriWJ** [verbb/formie#1176]
### Describe the bug

I’m setting up the Monday integration on one of my forms, but the board I need doesn’t appear in the form settings. The board drop-down menu in the form settings only displays 28 results. I think this may have something to do with the Monday API’s default limit for the number of boards. Could you let me know what I’d need to do to get all the Monday boards to display in the form settings?

### Steps to reproduce

1.  Add the Monday API key to the Formie plugin settings.
2. Go into the form settings and select "Integrations" and then "Monday".
3. Select the drop-down menu labelled "Board" and I can only see 28 results instead of all the boards on the Monday account.

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

3.7.13

### Plugin version

1.5.15

### Multi-site?

No

**Comment by LoriWJ** [comment]
I have some new information: The API appears to be pulling actions taken on the Monday account and not the actual boards on the Monday account.

**Comment by engram-design** [comment]
How many boards in your account do you have, out of interest? We are [querying](https://github.com/verbb/formie/blob/5dcde500398fdd1f0f138503d12593b3e28ab748/src/integrations/miscellaneous/Monday.php#L78-L101) boards, and I don't think it should be pulling in actions.

But does look like you're right that [25](https://developer.monday.com/api-reference/docs/boards) is the default limit. Would bumping that up to 100 suite your needs, or will you need more than that? Is so, we'll need to paginate results.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.18",
  "...": "..."
}
```

Then run `composer update`.

**Comment by LoriWJ** [comment]
Thanks Josh, that's great. I'll need to update Craft before I can get this version working, so can't test it right now.

**Comment by engram-design** [comment]
No problem, I've upped the limit to 100, and also added pagination, so that should hopefully solve it all. Let me know whenever you get to it and we can review if still not working.

**Comment by engram-design** [comment]
Fixed in [2.0.18](https://github.com/verbb/formie/releases/tag/2.0.18) and [1.6.19](https://github.com/verbb/formie/releases/tag/1.6.19)
